browser/src/DownloadsPage/GnomadV2Downloads.tsx

```diff
@@ -284,6 +286,29 @@ const GnomadV2Downloads = () => {
         </ColumnsWrapper>
       </DownloadsSection>
 
+      <DownloadsSection>
+        <SectionTitle id="v2-browser-tables">Browser Tables</SectionTitle>
+
+        <p>
+          In addition to our variants tables, we release two data tables underlying the gnomAD
+          browser. These tables enable our users to more easily incorporate gnomAD data into
+          external pipelines and analyses in a manner consistent with what they see in the browser.
+          For more information, see our
+          <Link to="/help/v4-browser-hts">help text</Link>
+        </p>
+
+        <h3>Browser gene models</h3>
```

Comment:
naive question, does a browser variants for v2 exist?

Reply:
Ah good question, it does yes, every gnomAD release of variants also has a browser version. We could release the v2 browser variants table too if we think that would be useful to the wider community.

Reply:
ah, OK. I vote we stick to releasing only the v4 version for now and seeing if there is demand for the v2 table